src/components/hotelCard.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { NavLink } from 'react-router-dom';
+import "../assets/styles/hotelCard.css";
+
+export default function HotelCard(props) {
```

Comment:
- [ ] In the hotel cards, there is a field for "Size", but it contains no info.  Kindly either add the correct "size" info or remove the field.  That way, we ensure consistency. 👍 

![image](https://user-images.githubusercontent.com/57421823/215270846-06cb3dbb-de38-4308-a144-f44ad3941ec7.png)